policyengine_us/parameters/gov/states/vt/tax/income/credits/ctc/age_threshold.yaml

```diff
@@ -0,0 +1,10 @@
+description: Vermont qualifies children under the Child Tax Credit below this age threshold.
+values:
+  2021-01-01: 0
+  2022-01-01: 6
```

Comment:
```suggestion
  2022-01-01: 5
```

Reply:
law specifies 5 or younger

Reply:
When I used 5 it didn't include 5 so it broke.

Reply:
Should I change it to 5, fix the description as less than equal to instead of under, and python file to check =< instead of <?.